### ISSUE TITLE
关于自部署网页版简悦的方法https://simp.red/trans?url=

### ISSUE THREAD

**Issue by sche11** [Kenshin/simpread#4896]
嗨，我注意到简悦曾经有个网页版的服务，部署在[vercel](https://simp.red/)上，这很棒，只需要在需要阅读模式的网页前加一段https://simp.red/trans?url=？，就可以在不支持拓展的手机浏览器上体验到不受打扰的阅读体验。
但遗憾的是，部署端失效了，我希望能够自己部署网页版的简悦，用于在rss阅读器中替换网址，来达到统一的阅读体验。
尽管有些rss阅读器提供全文解析，但对原文的破坏性太强，而简悦能够在保留原文阅读的基础上，剔除多余的干扰；
以及使用fv制作的自定义任务，在原文标注、剪切、同步到obsidian、flomo中，以及更多的拓展适应方式。

**Comment by Kenshin** [comment]
嗨，这个服务暂时下线了，之后会考虑重新上线的。

下线的其中一个原因是因为：这种方式对抓取动态加载的页面效果不是很好。

> 使用fv制作的自定义任务，在原文标注、剪切、同步到obsidian、flomo中，以及更多的拓展适应方式。

`fv` 是什么？以及 `原文标注、剪切、同步到obsidian、flomo中，以及更多的拓展适应方式。` 具体指什么？

**Comment by sche11** [comment]
> 嗨，这个服务暂时下线了，之后会考虑重新上线的。
> 
> 下线的其中一个原因是因为：这种方式对抓取动态加载的页面效果不是很好。
这点不需要担心,大多数博客文章都使用静态, 动态加载页面可以发送到剪切板,等回到电脑桌再使用简悦阅读,这部分少所以可以集中处理.
> `fv` 是什么？
fv指[fv悬浮球](https://www.coolapk.com/apk/com.fooview.android.fooview),是一个在顶端显示的浏览器,轻量,随用随搜,参考[官方演示视频],他的自定义任务类似[quicker](http://getquicker.net/)可以实现块的操作,是简化版的[tasker](http://getquicker.net/),(https://m.youku.com/video/id_XMTU0NTMyMTEzMg==.html).
>以及 `原文标注、剪切、同步到obsidian、flomo中，以及更多的拓展适应方式。` 具体指什么？
一种示例,视频演示[Obsidian 从本地到云端](https://www.bilibili.com/video/av390875359/?p=1&vd_source=570248c09e03c17de786013eed754264),相关文档[微信到 Obsidian 2.0](https://www.ftls.xyz/posts/wechat2s32/)和[flask 使用及开发说明](https://gitee.com/kkbt/obsidian-csapi#flask-%E4%BD%BF%E7%94%A8%E5%8F%8A%E5%BC%80%E5%8F%91%E8%AF%B4%E6%98%8E)

**Comment by Kenshin** [comment]
> https://www.bilibili.com/video/av390875359

这是你的 B站账户吗？

> 动态加载页面可以发送到剪切板

我的意思是：如果使用了简悦的在线版方案，如果是动态的页面就无法成功获取到全文。（在这种情况，你有什么好办法？

PS：在线版简悦确实是接下来重启的计划之一。

**Comment by sche11** [comment]
> 这是你的 B站账户吗？

不是,这是我在obsidian中文社区看到的,原地址[Obsidian 从本地到云端](https://forum-zh.obsidian.md/t/topic/13145),这给我启发,
加上我的重心转向使用手机端的rss阅读器(手机的尺寸很适合阅读),而

> 尽管有些rss阅读器提供全文解析，但对原文的破坏性太强，而简悦能够在保留原文阅读的基础上，剔除多余的干扰

这给了我契机

> 如果是动态的页面就无法成功获取到全文。（在这种情况，你有什么好办法？

我在查询[huginn](https://github.com/huginn/huginn)的使用教程中有遇到过,地址[使用Huginn打造自动化信息处理中心](https://blog.csdn.net/unreliable_narrator/article/details/124309672)

>如果需要抓取动态网页（大型网站有一些用的是动态，普通网站还是静态为主），其实就在以下步骤之前多一步，用[Phantom Js Cloud](https://phantomjscloud.com/) Agent渲染动态网页成静态页面后接下文的步骤即可

这里也有[页面静态化](https://www.cnblogs.com/minqiliang/p/16695963.html),

**Comment by Kenshin** [comment]
动态化的抓取很多方式，我讨论的并不是技术上的问题，这种方式是有弊端的：

1. 并不是很理想，因为诉求不一样，这种方式类似爬虫，并不太需要精准的完全性页面结构。

2. 会相当消耗后端服务性能。

当然，只是就事论事，并不是每个页面都需要动态抓取。

PS：如果使用 Android 手机的话，为什么不直接使用 Kiwi Browser （这是可以完整运行简悦的浏览器）？

**Comment by sche11** [comment]
> 如果使用 Android 手机的话，为什么不直接使用 Kiwi Browser （这是可以完整运行简悦的浏览器）？

我也有在使用kiwi浏览器,工具不嫌多,好用即可,但主要的原因:

>尽管有些rss阅读器提供全文解析，但对原文的破坏性太强，而简悦能够在保留原文阅读的基础上，剔除多余的干扰

我需要在rss阅读器上使用,只要替换网址就可以实现保留原文排版的阅读,一些阅读器的全文解析或丢失部分正文,或代码乱行,或图片排版错乱,原网址阅读又有太多干扰,甚至还不停的提示下载app.

少量的文章我还可以使用kiwi浏览器打开,点击简悦的图标,但rss订阅的需要一定阅读太多了,不停的跳转会有割裂感.

**Comment by Kenshin** [comment]
okay

大概了解了你的诉求。

***

使用在线版也是需要跳到浏览器，然后浏览器打开 `https://simp.red/trans?url=xxxx` 后才能使用的，所以你怎么解决

> 不停的跳转会有割裂感.

这个问题？

**Comment by sche11** [comment]
>使用在线版也是需要跳到浏览器，然后浏览器打开 `https://simp.red/trans?url=xxxx` 后才能使用的，所以你怎么解决

rss阅读器可以使用内置浏览器打开

**Comment by Kenshin** [comment]
> rss阅读器可以使用内置浏览器打开，我只需要链接覆写即可

原来如此，有点意思...

可能要提高点这个工具的优先级了 😄
